src/client/Client.js

```diff
@@ -338,6 +339,18 @@ class Client extends BaseClient {
       .then(app => new ClientApplication(this, app));
   }
 
+  /**
+   * Tries to fetch the specified guild, if the provided guild isn't public, returns null
+   * @param {Snowflake} id ID of the public guild
```

Comment:
could this be changed to GuildResolvable, then use `this.guilds.resolveID` to resolve the ID

Reply:
Yes, it could. I've updated the JSDocs and the method itself.